src/pages/reference/compare-to-heroku.md

```diff
@@ -0,0 +1,93 @@
+Those moving from Heroku to Railway will find that we support CNCF Buildpacks just like Heroku does if you want to keep using their build solution.
+
+### Product
+
+We offer a single pane of glass for evolving your infrastructure over time. We are proud of the interfaces we provide, and our users report this is one of their favorite platform aspects.
+
+It's possible to go from zero → 3 services in <30 seconds. Adding new services trivially and meshing them together means you don't need to compromise on architecture because of your infrastructure.
+
+### Pricing
+
+Pay only for what you use—no need to size dynos.
+
+Containers scale vertically up to 8 GB/CPU Cores (on DevPlan) and 32 GB/CPU Cores (on Teams Plans), and you only pay per minute.
+
+Did traffic spike for an hour? Only pay for that hour, and no manual scaling intervention is required.
+
+For those who prefer to pay upfront. We offer credit-based Developer plans who wish to load their account with credits.
+
+### Develop and Deploy
+
+You can use the CLI to run every service locally, hooking in with your cloud infrastructure. Whenever you use
+e.g. `railway run yarn start` in a project with Postgres will automagically inject things like `DATABASE_URL` etc.
+
+If your changes look good, you can even run `railway up` from the terminal to trigger a new deployment on demand.
+
+### Lifecycle Management
+
+We offer guardrails to prevent new deployments from knocking your app offline, such as [Healthchecks](/diagnose/healthchecks). For example, by providing an endpoint for Railway to hit, we will make a series of requests, checking for a 200 OK before we switch deploys to the most recent one.
+
+Want to revert a change? Because we image all builds, we allow you to [rollback](/deploy/deployments#rollback) to an earlier working deploy in just one click.
+
+### Support + Community
+
+Railway serves tens of thousands of builders who deploy applications that bring tremendous impact to themselves, their friends, and their customers.
+
+We feel no project, big or small- is never not important. This is our guiding philosophy to our users.
+
+As a result, we make great pains to be very communicative in our support channels, [optimistically gather feedback](https://feedback.railway.app/), provide informative docs (this place), and enrich our community to help each other.
+
+[Join our community of 5k+ users on Discord who can help, and even chat directly with developers if the community can't solve your problem.](https://discord.gg/railway)
+
+## Moving From Heroku to Railway
+
+Getting your app hosted on Heroku moved over to Railway is very simple.
+
+All you need to do is connect your GitHub account, create a new project and then deploy your repo. We even support Procfiles out of the box. (Caveat: only the web worker process is supported for now)
```

Comment:
Nixpacks support all types of dyno (`web`, `worker`, `release`, custom ones too!).